pubspec.lock

```diff
@@ -1,885 +0,0 @@
-# Generated by pub
```

Comment:
same comment, if we were to keep the files this should be readded as well as package-lock.json.

Reply:
@apomalyn and for these files (package-lock.json & pubspec.lock) should we re-add those?

Reply:
I added the pubspec.lock but the package-lock and packages files are deprecated.